static/app/components/core/principles/motion/motion.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Motion
+layout: document
+description: Guidelines for using animation in Sentry's user interface.
+---
+
+import {MotionPlayground} from 'sentry/stories/playground/motion';
+
+Motion in the UI, when used thoughtfully, can bring interfaces to life and guide users through tasks. Effective animations provide feedback, reduce user frustration, and give a sense of direct manipulation of on-screen elements. They also make our product feel snappy and modern by conveying responsiveness. The following guidelines outline our motion design principles, best practices for using animation in common scenarios, and accessibility considerations.
+
+<MotionPlayground />
+
+# Principles
+
+## Purposeful
+
+Use animation only when it serves a clear purpose. Every motion should communicate something useful (e.g. feedback, a state change) and not exist just for decoration. If an animation doesn’t help the user achieve a goal or learn something, reconsider using it.
```

Comment:
This seems too restrictive. Are delightful and engaging animations (e.g. for product onboarding screens) counted as "useful"? If yes, it will be helpful to clarify what we mean by useful.